src/components/panel/panel.js

```diff
@@ -284,6 +284,15 @@ angular
  * toggled the class and animations are completed.
  */
 
+/**
+ * @ngdoc method
+ * @name MdPanelRef#focusOnOpen
+ * @description
+ * Focuses the panel content if the focusOnOpen config value is true. This should
```

Comment:
I wonder if we should call this method automatically in the addClass state (if focusOnOpen is true). My concern with having the user do it is that they will forget and then the panel will not be accessible.

I know you are using addClass in createPanel, but I think that executing the applyAsync and focus any time after compile should also work.